mapflow/mapflow.py

```diff
@@ -1525,7 +1525,7 @@ def set_processing_limit(self, response: QNetworkReply) -> None:
         if response_data.get('admin'):
             self.remaining_limit = 1e5  # 100K sq.km
         else:
-            self.remaining_limit = response_data.get('remainingArea', 0)
+            self.remaining_limit = int(response_data.get('remainingArea', 0)) / 1e6  # convert into sq.km
```

Comment:
what kind of rounding do we make here? It's better to show it explicitely. 

should we floor(), to make <1 sqkm -> 0?

Or maybe, leave 1-2 decimals?